src/components/quickStarts.stories.mdx

```diff
@@ -259,13 +259,13 @@ Result:
 
 ## Handle Device Selection Error
```

Comment:
I think we can just club the last and this section under "Handle Device Selection Error".

Reply:
I'm not following, what are you talking about?

Reply:
Discussed offline and we decided to remove this section because the device selection error can be handled in a straightforward way in V3.